packages/react-component-library/cypress/specs/forms/empty.ts

```diff
@@ -129,9 +104,13 @@ describe('Form Examples', () => {
               })
 
               it('should supply form the field values', () => {
-                cy.get(selectors.form.values).contains(
-                  JSON.stringify(expectedResult[name], null, 2)
-                )
+                cy.get(selectors.form.values)
+                  .invoke('text')
+                  .should((submittedData) => {
+                    expect(JSON.parse(submittedData)).to.deep.equal(
```

Comment:
Adjusted this to avoid having to maintain the three variations of the form values (only differing by sorting).

If a deep equal assertion fails, once the test has finished you can click on the `assert` row in Cypress to get the two objects in full so you can compare them:

![Screenshot from 2022-02-22 13-23-16](https://user-images.githubusercontent.com/66470099/155157381-148336af-620f-433a-a93b-f7084fa08e40.png)